src/masternode/masternode-meta.h

```diff
@@ -134,13 +136,32 @@ class CMasternodeMetaMan
         }
 
         READWRITE(nDsqCount);
+
+        if (ser_action.ForRead()) {
+            READWRITE(nCurrentVersion);
+            READWRITE(nCurrentVersionStarted);
+            if (nCurrentVersion != MIN_MASTERNODE_PROTO_VERSION) {
+                // serialized by previous version
+                nCurrentVersion = MIN_MASTERNODE_PROTO_VERSION;
+                nCurrentVersionStarted = GetTime();
+            }
+        } else {
+            if (nCurrentVersion == 0 && nCurrentVersionStarted == 0) {
+                // first time serialization
+                nCurrentVersion = MIN_MASTERNODE_PROTO_VERSION;
```

Comment:
Is it a good idea to update class variable in a serialization (write) operation? Should be const IMO, instead write the literals if first time serializing